### ISSUE TITLE
Campaign URL Not Showing Cloned Landing Page

### ISSUE THREAD

**Issue by kjohnson5** [gophish/gophish#888]
**Thanks for reaching out! We're happy to help resolve issues as quickly as possible.**

**Please use this template when creating a new issue. If you do not follow this template format, your issue may be closed without comment.**

Before filing a new issue, please use the search bar at the top of the browser to search for similar issues. Also, please make sure you have read our [documentation](https://getgophish.com/documentation/), which covers how to use Gophish.

What version of Gophish are you using?: 0.4

Brief description of the issue: I have changed the configuration so that the listening url is 0.0.0.0:3333. I have an external domain that I've setup to use for GoPhish. I've nat one of my public IP address to the server's IP address on ports 80, 443, and 3333, however, when I use the external site for the landing page, I am getting the "This site can't be reached" error. What am I missing to get this working?

![image](https://user-images.githubusercontent.com/34453644/33948533-15b27392-dff5-11e7-9e98-1cbf3febb6f0.png)

What are you expecting to see happen? : Expecting to see a clone of the landing page I specified

What are you seeing happen? Receiving error message "This site can't be reached"

If this question is related to email templates or landing pages not working as expected, please provide your template or landing page below:

```html
Insert code here
```

Please provide any terminal output that may be relevant below:

```
Insert output here
```

Please provide as many steps as you can to reproduce the problem:
 * Step 1 - Purchase an external domain
 * Step 2 - Set external domain to one of my external IP address
 * Step 3 - Nat the public IP address to the server's IP address
 * Step 4 - Verify DNS is working with MX toolbox
 * Step 5 - Set URL in Campaign to external domain

**Comment by jordan-wright** [comment]
Are you sure there isn't a firewall rule getting in the way? It sounds like it's almost certainly a networking problem. I'd try to test each part individually if possible:

* Can you resolve the domain correctly?
* Can you verify that connections are getting NAT'd correctly?
* Can you verify that you can hit any connection through that NAT?
* Can you verify that you can hit Gophish locally on the server?

That should help you narrow it down, but it sounds like the Gophish specific parts are setup correctly, so I'm afraid there's not much we can do here. Let me know if this helps at all.

**Comment by kjohnson5** [comment]
To answer your questions:

I am able to resolve the domain correctly. I've used MX Toolbox to ensure that the domain is associated with the correct IP address. The firewall on the server is disable. Whenever I enable the firewall on the server, I am unable to access the admin page for GoPhish. I've ruled out the corporate firewall by ensuring that the NATing is working by changing the NAT internal IP to another server and when I access the external domain, I am able to get to that server's website without issue. When the url is changed to the local sever IP address or FQDN then I am able to get the cloned landing page. I've never run into this issue before with NATing so I'm a bit lost as to where in the routing of traffic is causing the issue.
